src/routes.json

```diff
@@ -24,6 +24,18 @@
         "title": "Hour of Code"
     },
     {
+        "name": "explore",
+        "pattern": "^/explore/:projects/:all/?$",
```

Comment:
Yeah, good point, I think we could do a replacement like the one below to cover these so they would all become e.g. `^/explore/.*?/.*?/?$`, since it would be tough to configure the translation of the format of each argument from express to fastly.
```javascript
pattern.replace(/(:[^/]*)\//gi, ".*?/")
```